index.html

```diff
@@ -0,0 +1,107 @@
+  <link rel="stylesheet" href="stylesheets/mainpagestyle.css">
+  <title>Film Festival Chennai 2022</title>
+</head>
+
+<body>
+  <header id="control-nav-bars">
+    <a href="#" class="hamburger-wrapper"><img class="hamburger-icon" src="bg-icons/hamburger-icon.svg" alt="hamburger menu icon"></a>
+    <div class="control-bar">
+      <ul class="control-list no-style">
+        <li class="control-list-item"><a href="https://www.facebook.com" target="_blank" rel="noopener"><img class="social-logo" src="bg-icons/facebook-icon.svg" alt="Facebook icon"></a></li>
+        <li class="control-list-item"><a href="https://www.twitter.com" target="_blank" rel="noopener"><img class="social-logo" src="bg-icons/twitter-icon.svg" alt="Twitter icon"></a></li>
+        <li class="control-list-item"><a href="">English</a></li>
+        <li class="control-list-item"><a href="">MyPage</a></li>
+        <li class="control-list-item"><a href="">Logout</a></li>
+      </ul>
+    </div>
+    <div class="navigation-bar">
+      <a href="index.html"><img class="logo-image" src="logo-resources/navbar-logo.svg" alt="image of video camera and title"></a>
+      <ul class="navigation-list no-style">
+        <li class="navigation-list-item"><a class="orange-text" href="about.html">About</a></li>
+        <li class="navigation-list-item"><a href="">Schedule</a></li>
+        <li class="navigation-list-item"><a href="">Venues</a></li>
+        <li class="navigation-list-item"><a href="">Partner</a></li>
+        <li class="navigation-list-item"><a href="">News</a></li>
+        <li class="navigation-list-item orange-box"><a href="">Tickets</a></li>
+      </ul>
+    </div>
+  </header>
+  <section class="section-intro">
+    <small class="header-welcome">"Hello! Vanakkam"</small>
+    <h1 class="header-title">TAMIL FILM ASSOC.<br>FILM FESTIVAL<br>2022</h1>
+    <p class="intro-description">Most Popular Film Festival in South India, gathering film talents from all
+      over the World. More than
+      20 films are to be screened throughout the week-long festival. And much more. </p>
+    <h2 class="date-detail">14.11.2022(MON) ~ 20(SUN)</h2>
+    <p class="venue-detail">@ Chennai Trade Center,Nandambakkam</p>
+  </section>
+  <section class="section-program">
+    <h2 class="header-programs">Main Program</h2>
+    <div class="orange-line"></div>
+    <div class="program-wrapper">
+      <div class="program-tile">
+        <img src="bg-icons/movie-screen-icon.svg" alt="movie screening" class="program-icon">
+        <h4 class="program-title">Screenings</h4>
+        <p class="program-description">Over 20 films to be screened during the week</p>
+      </div>
+      <div class="program-tile">
+        <img src="bg-icons/workshop-icon.svg" alt="movie screening" class="program-icon">
+        <h4 class="program-title">Workshops</h4>
+        <p class="program-description">Workshops by eminent directors and technicians</p>
```

Comment:
- [ ] Kindly implement the `hover` properties of your cards to match the original design

![image](https://user-images.githubusercontent.com/98453979/210375373-34de2513-7077-48b2-8e27-33c96da9d6bd.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/210375587-8ff80662-244b-42f0-8099-6bb14a8ef9e2.png)